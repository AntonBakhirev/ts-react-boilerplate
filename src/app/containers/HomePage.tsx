import {color} from "csx";
import * as React from "react";
import {FormattedMessage} from "react-intl";
import {style} from "typestyle";

const styles = {
  home: style({
    color: color("#125588").fade("80%").toString(),
    textAlign: "center"
  })
};

class HomePage extends React.Component {
  public render(): JSX.Element {
    return (
      <div className={styles.home}>
        <img src={require("../images/barbar.png")}/>
        <p><FormattedMessage id="hello" defaultMessage="Hello!" /></p>
      </div>
    );
  }
}

export {HomePage};
