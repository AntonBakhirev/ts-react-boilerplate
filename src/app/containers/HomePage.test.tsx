// import { renderComponent } from "../helpers/TestHelper";
import {shallow} from "enzyme";
import * as React from "react";
import {HomePage} from "./HomePage";
describe("<HomePage />", () => {

  const component = shallow(<HomePage />);

  it("matches snapshot", () => {
    expect(component).toMatchSnapshot();
  });
});
