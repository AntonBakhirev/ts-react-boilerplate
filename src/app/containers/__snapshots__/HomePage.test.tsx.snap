// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> matches snapshot 1`] = `
<div
  className="fsuivmx"
>
  <img
    src={Object {}}
  />
  <p>
    <FormattedMessage
      defaultMessage="Hello!"
      id="hello"
      values={Object {}}
    />
  </p>
</div>
`;
